src/pages/dashboard/TagFiltering.tsx

```diff
@@ -15,9 +15,11 @@ import ErrorBanner from '../../components/ErrorBanner';
 
 const minimumTagLength = 1;
 
+const StyledAutocomplete: any = styled(Autocomplete)`
```

Comment:
hack for å motvirke at  material-ui setter fullwidth om man vil eller ikke

Reply:
nesten så jeg ville hatt denne inline

Reply:
Synes det er greit at den står der som styled-components, vi har jo masse plasser styled-components med kun en linje.